modal/app.py

```diff
@@ -879,15 +879,17 @@ def cls(
         if _warn_parentheses_missing:
             raise InvalidError("Did you forget parentheses? Suggestion: `@app.cls()`.")
 
+        # Argument validation
         if interactive:
             deprecation_error(
                 (2024, 5, 1), "interactive=True has been deprecated. Set MODAL_INTERACTIVE_FUNCTIONS=1 instead."
             )
 
-        if image is None:
-            image = self._get_default_image()
-
-        secrets = [*self._secrets, *secrets]
+        scheduler_placement = _experimental_scheduler_placement
```

Comment:
BTW should we deprecate `_experimental_scheduler_placement`? Is it supplanted by the stable `region=` API now?

Reply:
I don't believe so, since `_experimental_scheduler_placement` allows you to specify `zone`, `spot`, and `instance_type` in addition to `region`.